dora/core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2625,7 +2625,7 @@ public static String getSyncMetricName(long mountId) {
           .build();
 
   public static final MetricKey NETTY_CHANNEL_COUNT =
-      new Builder("Netty.NettyChannelCount")
+      new Builder("Process.NettyChannelCount")
```

Comment:
@JiamingMai PTAL, the original bin/alluxio docGen command complain about Netty is not one of the existing instance type
```
  public enum InstanceType {
    CLUSTER("Cluster"),
    SERVER("Server"),
    MASTER("Master"),
    WORKER("Worker"),
    SECURITY("Security"),
    JOB_MASTER("JobMaster"),
    JOB_WORKER("JobWorker"),
    PLUGIN("Plugin"),
    PROCESS("Process"),
    PROXY("Proxy"),
    CLIENT("Client"),
    FUSE("Fuse");

```